packages/gluestick/src/generator/generator.js

```diff
@@ -0,0 +1,76 @@
+/* @flow */
+
+import type {
+  Logger,
+  Generator,
+  WrittenTemplate,
+  GeneratorOptions,
+} from '../types';
+
+const path = require('path');
+
+const { isValidEntryPoint } = require('../generator/utils');
+const requireGenerator = require('./requireGenerator');
+const parseConfig = require('./parseConfig');
+const writeTemplate = require('./writeTemplate');
+
+type Command = {
+  generatorName: string;
+  entityName: string;
+  options?: GeneratorOptions;
+}
+
+/**
+ * Starts generator routine.
+ *
+ * @param command {Object} { generatorName, entityName, entryPoint, options } Command object
+ * @param logger {Object} logger Logger instance
+ */
+module.exports = (command: Command, logger: Logger): void => {
+  const {
+    generatorName,
+    entityName,
+    options = {},
+  } = command;
+  const {
+    entryPoint = '',
+  } = options;
+
+  const predefinedGenerators = [
+    'component',
+    'reducer',
+    'container',
+  ];
+
+  if (predefinedGenerators.find((element) => element === generatorName)) {
+    if (!isValidEntryPoint(entryPoint, logger)) {
+      return;
```

Comment:
Same comment as the previous `return`. `isValidEntryPoint` already logs errors + info.